### ISSUE TITLE
Mixpanel Server Error: 0

### ISSUE THREAD

**Issue by devshorts** [mixpanel/mixpanel-node#190]
I am getting this error semi frequently on a consistent event type and I can't seem to tell if there is something in the _payload_ that is causing this or if this is something else. Is there any insight to be found what server error 0 means?

**Comment by tdumitrescu** [comment]
You can initialize the library with `{verbose: true, debug: true}` and that should give some console logging of error messages from the API.

**Comment by devshorts** [comment]
I see, thanks for that. Unfortunately, I can't really repo this local and I am seeing this in production. I'd rather not enable this in production, is there no information about what this error code could mean so I could try and dig a little further?

**Comment by tdumitrescu** [comment]
Unfortunately not - in the non-verbose default API mode, "0" is a catch-all for any error.

**Comment by devshorts** [comment]
I was able to find the error in verbose mode, but for what its worth it would be much more helpful for an integration standpoint to just throw the error properly. Having 0 as a catch-all is useless and not helpful.

**Comment by ak--47** [comment]
@devshorts 

yes, this the design of the `/track` API ... nothing [older than 5 days](https://developer.mixpanel.com/reference/track-event#when-to-use-track-vs-import)

should you wish to import events older than 5 days, you can `init` the library with an [API secret ](https://help.mixpanel.com/hc/en-us/articles/115004490503-Project-Settings#api-secret) and then use the `.import()` method to send an older timestamp:

```javascript
const mixpanel = Mixpanel.init('{{token}}', {
    secret: ' {{api secret}}'
});

mixpanel.import('super old event', new Date(2019, 4, 20, 12, 34, 56), {
    distinct_id: 'ak',
});
```

if you don't know the 'age' of the event ahead of time, there is no downside to using `.import()` for everything.
